airflow/config_templates/default_airflow.cfg

```diff
@@ -170,6 +170,8 @@ default_ram = 512
 default_disk = 512
 default_gpus = 0
 
+# Default mapreduce queue for HiveOperator tasks
+default_hive_mapred_queue =
 
```

Comment:
This is a more generic config section that applies to all operators, this one should be moved to a new [hive] section.

Reply:
Done, moved it to a `[hive]` section.